collectors/feature/gitlab/src/main/java/com/capitalone/dashboard/collector/FeatureSettings.java

```diff
@@ -0,0 +1,59 @@
+package com.capitalone.dashboard.collector;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+/**
+ * Bean to hold settings specific to the Feature collector.
+ * 
+ */
+@Component
+@ConfigurationProperties(prefix = "gitlab")
```

Comment:
@rob-miller-777 @tabladrum If we use a different properties file name like "gitlabfeature.properties" and start the gitlabfeature collector, that should solve the issue too. The prefix within the gitlabfeature.properties could remain gitlab. What do you think ?